src/components/suggestions/suggestions.scss

```diff
@@ -1,3 +1,135 @@
+@use './suggestions.properties.scss';
+
 .lui-suggestions {
-  // component styles
+  border: var(--lui-suggestions-border);
```

Comment:
Add a background for this as well, otherwise, it is transparent by default. You can see it on Storybook on the Suggestions Docs page

Reply:
Done